pupil_src/shared_modules/background_helper.py

```diff
@@ -0,0 +1,60 @@
+'''
```

Comment:
I really like this abstraction. However I feel it could be more pythonic to make the tast_proxy a class with methods instead of fns with proxy_obj. The current implementation feels like C. 

Any reason not to make this a class?